folder_paths.py

```diff
@@ -139,6 +139,7 @@ def add_model_folder_path(folder_name: str, full_folder_path: str) -> None:
         folder_names_and_paths[folder_name] = ([full_folder_path], set())
 
 def get_folder_paths(folder_name: str) -> list[str]:
+    global folder_names_and_paths
```

Comment:
This isn't needed - https://stackoverflow.com/questions/11867359/when-do-i-need-to-use-the-global-keyword-in-python